js/index.js

```diff
@@ -40,3 +40,79 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navBar = document.querySelectorAll('a');
+let navB = Array.from(navBar);
```

Comment:
You could move your `Array.from()` to surround the `document.querySelectorAll('a')` to simplify it down to one line like this
```js
let navBar = Array.from(document.querySelectorAll('a'));
```